js/ui/windowManager.js

```diff
@@ -2136,6 +2136,12 @@ const WindowManager = new Lang.Class({
             }
 
             if (SideComponent.isSideComponentWindow(actor.meta_window) && this._showDesktopOnDestroyDone) {
+                if (!Main.appStore.appLaunched) {
+                    Main.overview.showApps();
+                }
```

Comment:
Ooops! I removed all those from appStore.js, but I clearly forget about this place. I'll fix that